### PR TITLE
Fixed Gh-pages

### PR DESCRIPTION
**Pull request by drillsar**
@DrChat I see you deleted buildbot to change it from not displaying on index page what do I change? I see it in _config.yml not sure what that does though.

### CODE REVIEW

File: development/quickstart.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ permalink: /development/quickstart/
 
 ## Getting Started
 
-Have Windows 8.1+ with Python 2.7 and [Visual Studio 2015](https://www.visualstudio.com/downloads/download-visual-studio-vs) and the Windows SDKs installed:
+Have Windows 8 or later with Python 3.6 and [Visual Studio 2017](https://www.visualstudio.com/downloads/download-visual-studio-vs) and the Windows SDKs installed:
```
Comment (gibbed) [inline development/quickstart.md]: Why are you changing this? 2017 isn't required to build Xenia.

File: development/quickstart.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ When pulling new code use either `xb pull` to fetch the latest changes to both
 the main repository and all submodules as well as run premake, or else be sure
 to do those things yourself.
 
-### Visual Studio 2015 Configuration
+### Visual Studio 2017 Configuration
```
Comment (gibbed) [inline development/quickstart.md]: Ditto.

File: development/quickstart.md (hunk 1 above)
Comment (drillsar) [inline development/quickstart.md]: Was a mistake will change it now

File: development/quickstart.md [hunk 3]
```diff
@@ -10,7 +10,7 @@ permalink: /development/quickstart/
 
 ## Getting Started
 
-Have Windows 8.1+ with Python 2.7 and [Visual Studio 2015](https://www.visualstudio.com/downloads/download-visual-studio-vs) and the Windows SDKs installed:
+Have Windows 8 or later with Python 3.6 and [Visual Studio 2015 or 2017](https://www.visualstudio.com/downloads/download-visual-studio-vs) and the Windows SDKs installed:
```
Comment (DrChat) [inline development/quickstart.md]: At the moment we require Python 2.x

Comment (drillsar) [inline development/quickstart.md]: I know, but the pull request for python 3 is in the making and works on my end:
Pull #755

Comment (drillsar) [inline development/quickstart.md]: Ok fixed that.

**Comment by spiringmovie** [comment]
how would we find roms to test without having to illegally bypass the copy protection on the disks

File: faq.md [hunk 4]
```diff
@@ -10,9 +10,7 @@ permalink: /faq/
 
 ## Can I get an exe?
 
-**NO**, not yet. Xenia is in its extremely early phases and not ready for use by
-non-developers. If you can't build it yourself using the instructions provided
-it probably means you should wait until alpha/beta/etc.
+Yes, by going to the latest build <a href="https://ci.appveyor.com/project/benvanik/xenia/build/artifacts">here</a> for Windows. From there go to Jobs > Configuration Release > Artifacts than you can dowload the zip file.
```
Comment (DrChat) [inline faq.md]: "[...] Release > Artifacts than you can [...]" Change "than" to "then"

Comment (drillsar) [inline faq.md]: OOps; fixed

Comment (drillsar) [reply faq.md]: @DrChat  Is this good to go?

**Pull request merged**